code/datums/greyscale/config_types/greyscale_configs.dm

```diff
@@ -277,3 +277,13 @@
 	name = "Gold Worn Cleric Mace"
 	icon_file = 'icons/obj/items/cleric_mace.dmi'
 	json_config = 'code/datums/greyscale/json_configs/items/cleric_mace_worn_gold.json'
+
+/datum/greyscale_config/sweater_gags
+	name = "turtleneck sweater"
+	icon_file = 'icons/obj/clothing/suits.dmi'
```

Comment:
You don't have to do this but I would greatly appreciate you taking the icons out of their current mess of a file and putting all the sweater icons into a single dmi.

Reply:
it's all so broken already I feel like this will break it more somehow and I'm at the point where I just want to finish this. I'm not a coder or anything I just thought sweaters would be cool and "it wouldn't be that hard right?" I was wrong.